doc/pymode.txt

```diff
@@ -353,7 +353,7 @@ Pymode has the ability to set code checkers options from pymode variables:
 Set PEP8 options                                   *'g:pymode_lint_options_pep8'*
 >
     let g:pymode_lint_options_pep8 =
-        \ {'max_line_length': g:pymode_options_max_line_length})
+        \ {'max_line_length': g:pymode_options_max_line_length}
```

Comment:
I had tried to copy & paste this line from the help doc into my ftplugin for python files, but was getting syntax errors b/c of the trailing ")"